packages/lisk-client/package.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
How about adding the other `package-lock.json`s at this point?

Reply:
in this branch, I think we are still at npm3 so no point to include `package-lock.json` now?